elements/pl-dataframe/pl-dataframe.py

```diff
@@ -0,0 +1,110 @@
+from typing import cast
+
+import chevron
+import lxml.html
+import pandas
+import prairielearn as pl
+
+NO_HIGHLIGHT_DEFAULT = False
+PREFIX_DEFAULT = ""
+SUFFIX_DEFAULT = ""
+TEXT_DEFAULT = False
+SHOW_HEADER_DEFAULT = True
+SHOW_INDEX_DEFAULT = True
+SHOW_DIMENSIONS_DEFAULT = True
+SHOW_DATATYPE_DEFAULT = False
+ADD_LINE_BREAKS_DEFAULT = False
+INTERACTIVE_DEFAULT = True
```

Comment:
I think the interactive DF table was paused due to the external dependency part. This might need to get deleted as a result.

Reply:
Ah this was left in by mistake, will remove.